pyproject.toml

```diff
@@ -34,18 +31,16 @@ dependencies = [
 	"marshmallow~=3.13.0",
 	"pywin32 ; platform_system=='Windows'",
 	"pytoml==0.1.21",
-	"PyYAML~=5.3 ; python_version<='3.9'",
-	"PyYAML~=6.0.1 ; python_version>='3.10'",
+	"PyYAML~=6.0.1",
```

Comment:
I think we should consider updating this

Reply:
concur. I can do this with the other in a bump python packages PR.